3-box-model/styles.css

```
@@ -0,0 +1,32 @@
+*{
```

```suggestion
body {
```
Avoid the use of  `*` it affects performance.